tests/admin/test_validator.py

```diff
@@ -41,6 +42,22 @@ def test_validate_email(self):
         eq_(response.detail, '"invalid_format" is not a valid email address.')
         eq_(response.status_code, 400)
 
+        # Two valid in a list
+        form = MultiDict([('help-email', valid), ('help-email', 'valid2@email.com')])
+        response = Validator().validate_email(Configuration.LIBRARY_SETTINGS, {"form": form})
+        eq_(response, None)
+
+        # One valid and one empty in a list
+        form = MultiDict([('help-email', valid), ('help-email', '')])
+        response = Validator().validate_email(Configuration.LIBRARY_SETTINGS, {"form": form})
+        eq_(response, None)
```

Comment:
What about situations in which everything in the list is required (i.e. an empty input should make the whole validation fail)...or does that never happen?

Reply:
I don't think there's a case when that would happen, since for a list you can have any number of items. The way the UI is, I think it's pretty common to leave the last one blank- at least, that's what I personally do. It's not clear if you have to click add on the last one for it to be included, so I do, and then I get an empty one that also gets submitted. The only problem I could think of was if empty string was actually a valid list item for some reason, but we don't have anything like that.